src/api/procedures/__tests__/createSecurityToken.ts

```diff
@@ -420,14 +482,20 @@ describe('createSecurityToken procedure', () => {
   });
 
   describe('getAuthorization', () => {
-    test('should return the appropriate roles and permissions', () => {
+    test('should return the appropriate roles and permissions', async () => {
+      entityMockUtils.getTickerReservationDetailsStub().resolves({
+        owner: entityMockUtils.getIdentityInstance(),
+        expiryDate: null,
+        status: TickerReservationStatus.Reserved,
+      });
+
       let proc = procedureMockUtils.getInstance<Params, SecurityToken, Storage>(mockContext, {
         customTypeData: null,
       });
 
       let boundFunc = getAuthorization.bind(proc);
 
-      expect(boundFunc(args)).toEqual({
+      expect(await boundFunc(args)).toEqual({
```

Comment:
In one of my PRs Jere told me to avoid inline awaits to help readability.

This is more readable than what I had, but I think its cleaner if this gets assigned to a result variable.

(comment applies to a few places here.)